### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by BobGu** [Leasw144/test1#1]
@Leasw144, hi thanks for you hard work on the assessment. Below I left feedback that I would like you to address before we consider you `technical ready`

~- [ ] It looks like you are missing your top down markdown portion of this assignment. Could you complete that so I can have a look?~

I actually found your top down design in another repo. I saw that this one is actually called `test1` which got me confused, it should be named `mod_0_skills`. Naming things is going to be super important moving forward. Could tell me a few reasons why naming things well might be important in programming, for yourself and for others?

- [ ] Both your `number-of-customer` and `current-staff-on-floor` functions are confusing.  How are you getting the number of staff, employees, or management? I see how you are getting the customer number, since there is an attribute.  Could you fix this?

- [ ] When we are thinking about how functions work, they should sound like verbs, and generally they modify an attribute in some way. So for example in `defrost` should a function that decreases the temperature. If the meat is initialized with a temperature of `38.9` then defrosting it should increase it 10 or so degrees. Could you modify any functions you are using to reflect this pattern that aren't current following it?

- [ ] Your `throw-out` function is a bit confusing. What does the meat temperature have to do with wether we should throw it out or not?

**Comment by Leasw144** [comment]
Hi Bob, yeah I got a little turned around during the assessment, I made two errors that put me back quite a bit and by that time I felt I was running the clock. The first if I recall was that I may have input the original command to add a remote from github incorrectly and didn't notice it. This would slow me down considerably later when it wouldn't push, and so the /test1 was me playing around with git until I eventually got it to work. However it worked when I wasn't expecting it to which is why it's under /test1 and not the intended URL. Additionally, I wasn't sure how to edit/delete url names or if that was even possible which added to the confusion.

I couldn't see how I'd input the command as well because I missed a bracket on another command which made Terminal not accept my commands so I had to restart the program. 

Naming things is important so as to avoid the exact situation that has occurred; there needs to be clear descriptions so that other people without the knowledge that we have, know what's going on and can easily reference important documents keep workflow running smoothly. It's also good for us personally to name things accurately as we may forget where a particular project is, having a clear title that explains the content prevents us from using any more mental energy than is necessary.

- For the numbers on staff, I wasn't sure if these numbers needed to be referenced or not when naming attributes so I opted out of it because I already had four listed. The methods and attributes have been changed to directly relate to attributes. I was caught up in this portion due to overthinking things and getting wrapped up in sub thoughts when I should have kept it simple. Let me know if the new values are still acceptable or if there's something I'm missing still.
- I'm a little unclear on what you mean when you say, "defrost should a function that decreases the temperature." However, I lowered the initial temperature of the meat, that way the action is to leave the meat outside until the temperature increases to 30 degrees. The wording on my part was a little misleading/unclear so I changed such that defrost modifies temperature
- This was a typo that I didn't catch because I'd spent so long on the earlier errors and figuring out what had happened. It should have read "(if meat was purchased 2 weeks after date-of-purchase, throw out)

So in other words, I may have gone a bit too fast, didn't play close attention to detail, and then was caught overthinking the Attribute/Methods portion. Additionally the delays made me not think as logically as I should have which lent itself to mistakes. In the future, I should probably try to pace myself a bit better and also not overthink things in other areas. I've updated the changes just now to this repo.

If possible, I would appreciate a do-over because I'm fairly confident I can get through it now, otherwise, let me know what I can do to be considered Turing ready. Thanks for the input!

**Comment by BobGu** [comment]
Hi thanks for your hard work and making those changes! I need you to address a second round of feedback before I can consider you `technical ready`

- [ ] For this function `current-occupancy (counts total number of people in restaurant)` how is the total count reached? Remember function need to use attribute(s) in someway or do something with them to arrive at a return value.

**Comment by Leasw144** [comment]
> Hi thanks for your hard work and making those changes! I need you to address a second round of feedback before I can consider you `technical ready`
> 
> * [ ]  For this function `current-occupancy (counts total number of people in restaurant)` how is the total count reached? Remember function need to use attribute(s) in someway or do something with them to arrive at a return value.

So I've changed my method to close-restaurant, the attribute customer is modified and set to 0 when the shop has been closed. I've made the changes, please let me know if this is acceptable or not, thanks
